.travis.yml

```diff
@@ -6,7 +6,7 @@ branches:
   - master
   - dev
 node_js:
-- '8'
+- '10'
```

Comment:
Ensure all the locations are changed - I don't know the exhaustive list.

Reply:
As far as I know it should be all of them. I did a search for 'node' and '8'.